examples/asr/notebooks/3_Speech_Commands_using_NeMo.ipynb

```diff
@@ -29,7 +29,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
```

Comment:
I disagree. I don't think notebooks should contain outputs.

Reply:
Hmm, good point. Alright I'll mark these two as resolved then.

Reply:
Do we have an agreement about output of notebooks? I would be happier to clean most of the output but leave some important results, say plots, to demonstrate to the user. But cleaning all output works for me either.

Reply:
For these notebooks, removing everything is fine. The issue comes up with the cad notebook 6. Downloading freesound dataset is very time consuming, and involves actively setting token values via a browser. 

In that case, leaving the plots might be beneficial so that the user has a rough idea of what's the end result without spending a long time building the dataset and training. @blisc what's your opinion regarding the vad notebook ?